packages/analytics/analytics-metric-provider/package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@kong-ui-public/analytics-utilities": "workspace:^0.8.4",
     "@kong-ui-public/core": "1.1.0",
-    "@kong-ui-public/metric-cards": "0.2.14",
+    "@kong-ui-public/metric-cards": "workspace:*",
```

Comment:
@adamdehaven The goal here was to be able to update both `metric-cards` and `analytics-metric-provider` in a single PR, without first needing to publish a new version of `metric-cards`.

I ran `add metric-cards@latest` and ended up with the `workspace:*` diff. Noticed that all other workspace version instances in this repo are `workspace:^`.  Safe to leave as-is, or use the hat here also?

Reply:
@adorack this sounds familiar to something we chatted about a long time ago -- do you remember? 😬

Reply:
<img width="1078" alt="Screenshot 2023-10-23 at 3 42 30 PM" src="https://github.com/Kong/public-ui-components/assets/103061463/b7f783ac-a732-4811-9a05-69d8ac913f6a">

Reply:
Sorry, I didn't see your ping.  (Working on fixing on-call issues from this weekend, so I haven't had time to comb through Github notifications.)  I don't have any additional information, unfortunately.